src/option.ts

```diff
@@ -18,6 +18,17 @@
 
   }
 
+  export declare namespace Document {
+    interface GetDocumentsParams {
+      projectId?: number[];
+      keyword?: string;
+      sort?: "created" | "updated";
+      order?: Order;
+      offset?: number;
```

Comment:
Let's make this required.

===

... To make it required would possibly make users feel weird because other params like `GetIssuesParams` don't require it.
But unlike `GetDocuments`, I found those endpoints work well even if `offset` isn't set.

Anyway `GetDocuments` requires `offset` 😬